rlclientlib/episodic_loop.cc

```diff
@@ -0,0 +1,47 @@
+#include "episodic_loop.h"
```

Comment:
Episodic api exposes std::vector and std::string through it's public interface.  Let's remove episodic api from the new interface till we can resolve this issue.

Reply:
removed episodic